src/@types/index.d.ts

```diff
@@ -5,3 +5,37 @@ interface Client {
 interface Fastly {
   client: Client;
 }
+
+interface Host {
+  host: string;
+  lookup: string;
+}
+
+interface Settings {
```

Comment:
As per other `apiKey` comment, the token now lives inside this interface a `token` property.